tests/types/test_string_type.cpp

```diff
@@ -427,30 +429,29 @@ TEST(StringType, Comparisons)
     */
 }
 
-/*
-TODO: Reenable this.
 
 TEST(StringType, Concatenation) {
     nd::array a, b, c;
 
+    nd::callable f = nd::functional::elwise(nd::callable::make<nd::string_concatenation_kernel>());
+
     a = "first";
```

Comment:
How you made the `nd::callable f` is also good. There's a way we declare these functions in DyND as global variables, which is all you are missing.